plutus-core/plutus-core/src/PlutusCore/Default/Builtins.hs

```diff
@@ -330,14 +328,17 @@ of them that are important to know to be able to use 'makeBuiltinMeaning' in cas
 complicated than a simple monomorphic or polymorphic function. But for now let's talk about a few
 more simple cases.
 
-4. Certain types are not built-in, but can be represented via built-in ones. For example, we don't
-have 'Int' as a built-in, but we have 'Integer' and we can represent the former in terms of the
+3. Certain types are not built-in, but can be represented via built-in ones. For example, we don't
+have 'Int' built-in, but we have 'Integer' and we can represent the former in terms of the
```

Comment:
> conceivable that we might add something like `Word8` or `Word64`

What for? We can already express those via `Integer` (and we already do for `Word8`) and it's unlikely that adding a new built-in type would help performance, since the cost of bounds checking that we currently do has to be virtually negligible compared to the cost of computing a builtin application. And a new built-in type is more code to write and maintain, plus potentially reduced performance due to GHC compiling big sum types less efficiently compared to small ones. Or do you have a use case in mind?

Reply:
> What for?

Who knows?  We have had requests for a full range of fixed-width types and operations in the past (although hopefully the bitwise thing will address that), so it's not out of the question that someone might request it in the future.  I was mostly just curious about what would actually happen though.